iib/workers/config.py

```diff
@@ -46,6 +46,8 @@ class Config(object):
         'iib.workers.tasks.build_create_empty_index',
         'iib.workers.tasks.general',
     ]
+    # Path to hidden location of SQLite database
+    hidden_index_db_path = '.iib/index.db'
```

Comment:
A thought: Is it wise to add 'hidden_path' to shared config? 
I would prefer to add an empty variable only.

Reply:
I mean anybody can extract the index image and see what is inside, right?

Reply:
@camilamacedo86 @gallettilance - I've suggested this path for `index.db`

Note: This is still in code review and things might change.

Reply:
Hey @gallettilance I was hoping to verify whether we want to keep this hidden path completely hidden from customers? I mean theoretically this path can still be seen by anyone from IIB source code. OR Is it sufficient that index.db is not directly available at older location now, so its semi-hidden

Reply:
it's sufficient for it not to be available at the old location

Reply:
the label on the image pointing to the index.db location should not be there anyway so OPM shouldn't be able to access it even if it were at the old location.